server/controllers/visit.js

```diff
@@ -8,14 +8,8 @@ var Boom = require('boom');
 
 var config = require('../config');
 var log = require('../logger')('server.controllers.visit');
-var url2png = require('url2png')(config.get('url2png_apiKey'), config.get('url2png_secretKey'));
 var visit = require('../models/visit');
-
-function addScreenshot(item) {
-  // TODO this is the seed of a view or view helper, believe it or not
-  item.screenshot_url = url2png.buildURL(item.url, {viewport: '1024x683', thumbnail_max_width: 540});
-  return item;
-}
+var view = require('../views/visit');
```

Comment:
Not sure if this should be something like `var visitView = ...;` for consistency with [`var visitsView = require('../views/visits');`](https://github.com/mozilla/chronicle/pull/359/files#diff-fc2554804aa3e40d22878ae59ff7baa9R15) below.